### DISCUSSION TITLE
How does opening new window work

### DISCUSSION THREAD

**Discussion by eshsdhsd** [FreeTubeApp/FreeTube#4865] [Q&A]
How does opening new window work because I get this for 5 seconds when opening a new window or opening free tube sometimes
![image](https://github.com/FreeTubeApp/FreeTube/assets/79751239/419fb43d-cb0e-44f7-85df-55c1e2bcc07c)

Is something loading? Does it load something then display or cant I chose to already display something at least
At least have the basic UI display as the highest priority
![image](https://github.com/FreeTubeApp/FreeTube/assets/79751239/8fabd10d-779a-4d09-8a6e-238da3979c81)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Opening another Instance of FreeTube could be considered the same as starting the application for the first time. So it behaves exactly the same the first time you started FreeTube.

**Comment by 4yman-0** [comment]
Can you please rename this to "new window freezes for 5 seconds"?
